resources/prosody-plugins/mod_muc_poltergeist.lua

```diff
@@ -409,9 +434,15 @@ function handle_create_poltergeist (event)
         return 202;
     else
         username = generate_uuid();
-        store_username(room, user_id, username);
+        local context = {
+            user_id = user_id;
+            group = group;
+            creator_user_id = session.jitsi_meet_context_user;
```

Comment:
jitsi_meet_context_user is actually the whole user object, not just a user id

Reply:
Yep, I was just renaming it :)